optin/reporting/test/controllers/completedController.spec.js

```diff
@@ -63,100 +70,93 @@ describe('completed controller', () => {
 			expect(result.symbolBalance).to.be.eql(mockSymbolSources.map(symbolInfo => symbolInfo.balance));
 		};
 
+		const runBasicQueryTests = async overWriteParams => {
+			// Arrange:
+			req = TestUtils.mockRequest({
+				...parameter,
+				...overWriteParams
+			});
+
+			getCompletedPaginationStub.returns(Promise.resolve([]));
+
+			// Act:
+			await CompletedController.getCompleted(req, res);
+
+			// Assert:
+			const result = res.json.getCall(0).firstArg;
+			expect(result).not.to.haveOwnProperty('error');
+		};
+
 		it('returns basic data', async () => {
-			const mockNemSources = [{
-				address: '682FAFBA20454869F0278748FD9790CFFCE35E8722647B4039',
-				balance: 79268194338335,
-				hashes: 'CDB0AF349823F1638E032DCAB15FD33049C383FEEE642CBC67E11020DC30F190',
-				height: 3004351,
-				label: 'Bithumb',
-				timestamps: '1609403596'
-			}];
-
-			const mockSymbolSources = [{
-				address: '688ADC5D31F49F918AC71DCE18E7085944AEAB0AF60F8AD7',
-				balance: 79268194338335,
-				hashes: 'E49B240D76DAE7277089C2BDA66B297A05AE700361ED253ED48435E9AF9B0FE1',
-				height: 1,
-				timestamps: 1615853185
-			}];
-
-			await runBasicCompletedDataTests(mockNemSources, mockSymbolSources);
+			await runBasicCompletedDataTests({
+				mockNemSources: mockCompletedData[0].nemSource,
```

Comment:
consider using enum or constant or object property instead of indexes as it will make tests easier to read

e.g. 
mockCompletedData.basic.nemSource
mockCompletedData.merge1.nemSource
mockCompletedData.merge2.nemSource
etc

Reply:
I see, that is good idea, i will update that 👍🏼